pages/common/tsv-filter.md

```diff
@@ -0,0 +1,36 @@
+# tsv-filter
+
+> Filter lines of a TSV file by running tests against individual fields.
+> More information: <https://github.com/eBay/tsv-utils#tsv-filter>.
+
+- Output all lines where column with field_name is numerically equal to value:
+
+`tsv-filter -H --eq {{field_name}}:{{number}} {{path/to/tsv_file}}`
+
+- Filter using numeric equality/non-equality/less-than/less-or-equal/greater-than/greater-or-equal:
```

Comment:
Based on the `[` page:
```suggestion
- Print the lines where a specific column is [eq]ual/[n]on [e]qual/[l]ess [t]han/[l]ess than or [e]qual/[g]reater [t]han/[g]reater than or [e]qual to a given number:
```
The example descriptions shouldn't be described using the context. Make them independent